exec/Chainweb/Backfill.hs

```diff
@@ -23,11 +25,14 @@ import           Database.Beam.Postgres (Connection, runBeamPostgres)
 ---
 
 backfill :: Env -> IO ()
-backfill e@(Env _ c _ _) = withPool c $ \pool -> do
+backfill e@(Env _ c _ _ cids) = withPool c $ \pool -> do
   putStrLn "Backfilling..."
   cont <- newIORef 0
-  mins <- minHeights pool
-  traverseConcurrently_ Par' (f pool cont) $ lookupPlan mins
+  mins <- minHeights cids pool
+  let !count = M.size mins
+  printf "Valid Chains: %d\n" count
```

Comment:
Let's change this to display a more informative error message to the user.  Or, better yet...instead of requiring that the user wait until all chains are present, can we use the chain list that we are now retrieving from the node?

Reply:
(1) Yes, let's change the message.
(2) Those are slightly orthogonal things. `mins` is the list of minimum heights as determined by looking at the DB. We need some definite starting point for each chain, and it might be dangerous to guess

Reply:
Ahh yes, I had a vague realization there might be something else needed.  A change to the message would be fine then.